src/SFA.DAS.ProviderApprenticeshipsService.Web/Helpers/HtmlExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+using System.Web;
+using System.Web.Mvc;
+using SFA.DAS.ProviderApprenticeshipsService.Domain.Models.FeatureToggles;
+using SFA.DAS.ProviderApprenticeshipsService.Web.Extensions;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.Helpers
+{
+    public static class HtmlExtensions
+    {
+        public static bool IsManageReservationsEnabled(this HtmlHelper htmlHelper, long providerId)
+        {
+            var service = DependencyResolver.Current.GetService<IFeatureToggleService>();
+            var isEnabled = service.Get<ManageReservations>().FeatureEnabled;
+            return isEnabled;
+        }
+
+        public static bool CanShowReservationsLink(this HtmlHelper htmlHelper)
+        {
```

Comment:
This could include a call to IsManageReservationsEnabled as this can never be true if enabled is false and currently the code makes both calls together.

Reply:
Addressed in https://github.com/SkillsFundingAgency/das-providerapprenticeshipsservice/pull/433/commits/6eb68540076759e6708b99b0d084f2701ec7f68f